tests/test_other.py

```diff
@@ -961,7 +961,7 @@ def test_wl_linkflags(self):
     -lfoo
     ''')
     run_process([PYTHON, EMCC, '-o', 'libfile.o', 'libfile.cpp'])
-    run_process([PYTHON, EMAR, 'rv', 'libfoo.a', 'libfile.o'])
+    run_process([PYTHON, EMAR, 'cr', 'libfoo.a', 'libfile.o'])
```

Comment:
the changes in this file seem separate? (I'm not opposed to including them, though.)

But what does this specific one do?